app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app import errors
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise errors.NotVaccinatedError(
+                "Everyone should be vaccinated")
```

Comment:
```suggestion
            raise errors.NotVaccinatedError(
                "Everyone should be vaccinated"
                )
```

The same in other similar cases